src/modeler/AutoRest.Swagger/Model/ValidationUtilities.cs

```diff
@@ -13,8 +13,14 @@ namespace AutoRest.Swagger.Model.Utilities
 {
     public static class ValidationUtilities
     {
-        private static readonly string XmsPageable = "x-ms-pageable";
+        private const string XmsPageable = "x-ms-pageable";
         private static readonly Regex TrackedResRegEx = new Regex(@".+/Resource$", RegexOptions.IgnoreCase);
+        private const string AzureHostSignature = "management.azure.com";
+
+        // if x-ms-parameterized-host extension is used or if the host is anything of the form management.azure, assume this to be an ARMServiceDefinition
+        // TODO: refine this definition for azure data plain also check x-ms-parameterized-host in detail to ensure this is ARM
+        public static bool IsARMServiceDefinition(ServiceDefinition serviceDefinition) => 
```

Comment:
what I really wanna say is: don't start anything like this before syncing with everyone - I suspect most rules are ARM specific (not also that we already classify the validation rules, so it'll be super easy to filter them *centralized* at some point, *not* from within the validation rules as you do here)